app/models/concerns/hyrax/collection_behavior.rb

```diff
@@ -19,13 +19,13 @@ module CollectionBehavior
     # Add members using the members association.
     def add_members(new_member_ids)
       return if new_member_ids.blank?
-      members << ActiveFedora::Base.find(new_member_ids)
+      members << find_resource(new_member_ids)
```

Comment:
I'm not sure that valkyrie's `find_by(id:)` can take/return an array.

Reply:
It can't, but there's also no reason to find the resource if you have the member IDs. Just add the member IDs.

Reply:
so switch this to `member_ids << new_member_ids`